cmd/gossamer/config.go

```diff
@@ -625,10 +619,10 @@ func setDotCoreConfig(ctx *cli.Context, tomlCfg ctoml.CoreConfig, cfg *dot.CoreC
 	case wasmer.Name:
 		cfg.WasmInterpreter = wasmer.Name
 	case "":
-		cfg.WasmInterpreter = gssmr.DefaultWasmInterpreter
+		cfg.WasmInterpreter = wasmer.Name
 	default:
-		cfg.WasmInterpreter = gssmr.DefaultWasmInterpreter
-		logger.Warn("invalid wasm interpreter set in config, defaulting to " + gssmr.DefaultWasmInterpreter)
+		cfg.WasmInterpreter = wasmer.Name
+		logger.Warn("invalid wasm interpreter set in config, defaulting to " + wasmer.Name)
```

Comment:
nit/a bit OOS: we should error or panic (maybe if this isn't user input error) here

Reply:
I mean do we really need to? All cases just end up using wasmer anyways. IMO even if its bad input since we still end up using the intended interpreter a log is fine, but let me know if you think otherwise

Reply:
We should remove this if we don't plan to use something else.  If we are going to investigate `wazero` we should leave this in.